packages/amplication-client/src/Resource/create-resource/wizard-pages/CreateServiceAuth.tsx

```diff
@@ -1,13 +1,52 @@
-import React, { useCallback } from "react";
+import React, { useCallback, useEffect, useMemo } from "react";
+import { PluginLogo } from "../../../Plugins/PluginLogo";
+import usePlugins from "../../../Plugins/hooks/usePlugins";
+import { EnumCodeGenerator } from "../../../models";
 import "../CreateServiceWizard.scss";
 import { CreateServiceWizardLayout as Layout } from "../CreateServiceWizardLayout";
 import { LabelDescriptionSelector } from "./LabelDescriptionSelector";
 import { WizardStepProps } from "./interfaces";
-import authModuleImage from "../../../assets/images/auth-module.svg";
-import ImgSvg from "./ImgSvg";
+
+const NODE_AUTH_PLUGINS = ["auth-jwt", "auth-basic", "auth-keycloak"];
+const DOTNET_AUTH_PLUGINS = ["dotnet-auth-jwt"];
+
+const OVERRIDE_PLUGIN_DESCRIPTION = {
+  "auth-jwt": "Use JSON Web Token (JWT) authentication",
+  "auth-basic": "Use basic (username-password) authentication",
+  "auth-keycloak": "Use Keycloak authentication",
+  "dotnet-auth-jwt": "Use JSON Web Token (JWT) authentication",
```

Comment:
Let's revisit this description with @overbit tomorrow to describe the right auth scheme we will use in dotnet

Reply:
Let's merge first, and anyway we need to update the plugin ID once it is ready (unless we already know what it is )